backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/Context.kt

```diff
@@ -209,6 +209,8 @@ internal class Context(config: KonanConfig) : KonanBackendContext(config) {
 
     override val configuration get() = config.configuration
 
+    override val internalPackageFqn: FqName = FqName.fromSegments(listOf("kotlin", "native", "internal"))
```

Comment:
Why not in https://github.com/JetBrains/kotlin-native/blob/master/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/RuntimeNames.kt?